src/Elastic.Apm.AspNetCore/Elastic.Apm.AspNetCore.csproj

```diff
@@ -9,6 +9,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\Elastic.Apm.EntityFrameworkCore\Elastic.Apm.EntityFrameworkCore.csproj" />

```

Comment:
This pulls in `Microsoft.EntityFrameworkCore.Relational` as transient dependency which might not go down well for folks who use asp .net core without EF.

Reply:
Yes, I was also thinking about this and that is a valid scenario. On the other hand the [Microsoft.AspNetCore.All](https://www.nuget.org/packages/Microsoft.AspNetCore.All) package already references EF Core, so I think most if the users already have that. And with this the benefit is that users don't have to explicitly enable the EF Core listener, which I think should be enabled by default. So the negative side of the change: for a small portion of users we add an unnecessary reference, the upside of the change is that for most of the users we avoid wondering how to enable db monitoring.

Reply:
Maybe it would be possible to also create additional csproj/nuget package `Elastic.Apm.All` that would contain helper that would add all subscribers and by default keep references to minimum. At least in case of running netcore apps in docker it is nice to have them small as possible

Reply:
This is a good idea!

Reply:
I implemented the suggestion from @lafriks, I think that's a clean way to handle this. Also updated the description.